### DISCUSSION TITLE
The support ticket does not have been replied for 8 days.

### DISCUSSION THREAD

**Discussion by sysop-88** [community/community#128265] [New to GitHub]
I made a support ticket which does not have been replied for **8 days**.  Where should I contact to get the support? Does the support ticket system down or they are just not answering? Please reply.

**Comment by AVMiraga** [comment]
I have never personally used a support ticket, but from what I've seen, there can be a significant wait time. It may take a month or two to receive a response, you will get an answer within that time range.

**Comment by sysop-88** [comment]
I need to reset **2FA** for **GitHub** log in. I changed my computer and lost **2FA
recovery**, so I need to remove **2FA** to log in and reset the password by contacting the support team.
Is there anyway I can contact the support team?

**Reply by AVMiraga** [reply]
Check out [this page](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). It states that if you've lost access to your 2FA, they won't be able to recover your account for you. Follow the recovery process mentioned in the provided link. If you are unable to recover it, unfortunately, there's no need to wait for the support team.

> For security reasons, GitHub Support [will not be able to restore access to accounts](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy) with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods.

also:

> If you cannot use any recovery methods, you have permanently lost access to your account. However, you can unlink an email address tied to the locked account. The unlinked email address can then be linked to a new or existing account. For more information, see "[Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account)."

**Comment by ghostinhershell** [comment]
Hi @sysop-88,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by sysop-88** [reply]
The GitHub page [Compare Plan](https://github.com/settings/billing/plans) shows that **'Standard Support - GitHub Support can help you troubleshoot issues you run into while using GitHub. Get support via the web.'** is not supported on **Free accounts**. Does that mean I almost do not have supports for **Free account**? I already used a support 2 weeks ago, then I need support for the other account.

**Comment by sysop-88** [comment]
The **GitHub** team can not recover account with **2FA**. So, if they remove **2FA**
for the account, they can help restore the account.
